### PR TITLE
Fixed: Provider Wiki Links (supported-X => supported#X)

### PR DESCRIPTION
**Pull request by bakerboy448**
[skip downstream]

#### Database Migration
NO

#### Description
fixup wiki link

#### Todos
-  Tests
-  Wiki Updates

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/Sonarr.Api.V3/ProviderResource.cs [hunk 1]
```diff
@@ -38,8 +38,9 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Message = definition.Message,
                 Tags = definition.Tags,
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
-
-                InfoLink = string.Format("https://wiki.servarr.com/Sonarr_Supported_{0}",
+                
+                //sonarrFixed: Provider Wiki Links (supported-X => supported#X)/supported is an disambagation page. the # should be a header on the page with appropiate details/link
```
Comment (markus101) [inline src/Sonarr.Api.V3/ProviderResource.cs:42]: Unnecessary comment?

**Review by markus101 — CHANGES_REQUESTED** [review]
I don't understand the comment here.

File: src/Sonarr.Api.V3/ProviderResource.cs (hunk 1 above)
Comment (bakerboy448) [reply src/Sonarr.Api.V3/ProviderResource.cs]: corrected up the comment
`// sonarr/supported is an disambiguation page. the # should be a header on the page with appropriate details/link`

**Comment by markus101** [comment]
Landed this fix in 0a30735f348253269b18a0329c46449c240d00f5

**Pull request closed without merging**